.github/workflows/qe.yml

```diff
@@ -29,17 +29,17 @@ jobs:
       SKIP_PRELOAD_IMAGES: true # Not needed for github-hosted runs
 
     steps:
-      - name: Free Disk Space (Ubuntu)
-        uses: jlumbroso/free-disk-space@main
-        with:
-          tool-cache: false
-          large-packages: true
-          android: true
-          dotnet: true
-          haskell: true
-          docker-images: true
-          swap-storage: true
-        continue-on-error: true
+      # - name: Free Disk Space (Ubuntu)
```

Comment:
We have a Git history; there is no need for commented-out code.

Reply:
I'll remove it once it actually works I think.  I'm not convinced the changes are fully rolled out yet.

Reply:
Done.